calendar-scheduling/src/test/java/com/linagora/calendar/scheduling/AlarmTriggerServiceTest.java

```diff
@@ -0,0 +1,675 @@
+            END:VEVENT
+            END:VCALENDAR
+            """);
+        alarmEventDAO.create(event).block();
+
+        testee.triggerAlarms().block();
+
+        // Wait for the mail to be received via mock SMTP
+        awaitAtMost.atMost(Duration.ofSeconds(20))
+            .untilAsserted(() -> assertThat(smtpMailsResponse().getList("")).hasSize(1));
+
+        JsonPath smtpMailsResponse = smtpMailsResponse();
+
+        assertSoftly(Throwing.consumer(softly -> {
+            softly.assertThat(smtpMailsResponse.getString("[0].from")).isEqualTo("no-reply@openpaas.org");
+            softly.assertThat(smtpMailsResponse.getString("[0].recipients[0].address")).isEqualTo("attendee@abc.com");
+            softly.assertThat(smtpMailsResponse.getString("[0].message"))
+                .contains("Subject: Notification: Alarm Test Event")
+                .contains("Content-Type: multipart/mixed;")
+                .containsIgnoringNewLines("""
+                    Content-Transfer-Encoding: base64
+                    Content-Type: text/html; charset=UTF-8
+                    """);
+
+            String html = getHtml(smtpMailsResponse);
+
+            softly.assertThat(html).contains("This event is about to begin in 10 minutes")
+                .contains("Test Room")
+                .contains("organizer@abc.com")
+                .contains("attendee@abc.com")
+                .contains("This is a test alarm event.");
+        }));
+    }
+
+    @Test
+    void shouldDeleteAlarmEventAfterSendingAlarmEmail() throws AddressException {
+        Instant now = clock.instant();
+        EventUid eventUid = new EventUid("event-uid-1");
+        AlarmEvent event = new AlarmEvent(
+            eventUid,
+            now.minusSeconds(10),
+            now.plusSeconds(3600),
+            NO_RECURRING,
+            Optional.empty(),
+            new MailAddress("attendee@abc.com"),
+            """
+            BEGIN:VCALENDAR
+            VERSION:2.0
+            BEGIN:VEVENT
+            UID:{eventUid}
+            DTSTART:20250801T100000Z
+            DTEND:20250801T110000Z
+            SUMMARY:Alarm Test Event
+            LOCATION:Test Room
+            DESCRIPTION:This is a test alarm event.
+            ORGANIZER;CN=Test Organizer:mailto:organizer@abc.com
+            ATTENDEE;CN=Test Attendee:mailto:attendee@abc.com
+            ATTENDEE;PARTSTAT=ACCEPTED;RSVP=FALSE;ROLE=CHAIR;CUTYPE=INDIVIDUAL:mailto:organizer@abc.com
+            END:VEVENT
+            END:VCALENDAR
+            """.replace("{eventUid}", eventUid.value()));
+        alarmEventDAO.create(event).block();
+
+        testee.triggerAlarms().block();
+
+        assertThat(alarmEventDAO.find(eventUid, new MailAddress("attendee@abc.com")).blockOptional()).isEmpty();
+    }
+
+    @Test
+    void shouldSendAlarmEmailWithHourDurationWhenDifferenceBetweenNowAndEventStartTimeIsGreaterThanOneHour() throws AddressException {
```

Comment:
This test case verifies the formatting of a email message content, 

It should be a unit test of static method `com.linagora.calendar.scheduling.AlarmTriggerService#formatDuration`